runelite-client/src/main/java/net/runelite/client/plugins/bankvalue/BankValueConfig.java

```diff
@@ -49,4 +49,13 @@ default boolean showHA()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "showExact",
+		name = "Show exact bank value",
+		description = "Show exact bank value")
```

Comment:
`)` on new line.

Reply:
I did think it was a bit weird, but it was what all the other configs were doing, so I just went with it. I'll fix the already existing configs too

Reply:
Strange... Must be whoever added this plugin formatted all of them that way. :man_shrugging: